autosklearn/util/data.py

```diff
@@ -198,6 +191,8 @@ def subsample(
         )
         left_idxs, _ = next(splitter.split(X=X, y=y))
 
+        # This function supports pandas objects but they won't get here
+        # yet as we do not reduce the size of pandas dataframes.
```

Comment:
Wait, we do not subsample pandas dataframes? I thought we subsample them, but do not reduce precision on them.

Reply:
See above comment on memory